commands/orchestrationcommands/stackcommands/adopt.go

```diff
@@ -0,0 +1,159 @@
+package stackcommands
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	osStacks "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stacks"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stacks"
+	"github.com/rackspace/rack/util"
+)
+
+var adopt = cli.Command{
+	Name:        "adopt",
+	Usage:       util.Usage(commandPrefix, "adopt", "--name <stackName> --adopt-file <adoptFile>"),
+	Description: "Creates a stack from existing resources",
+	Action:      actionAdopt,
+	Flags:       commandoptions.CommandFlags(flagsAdopt, nil),
+	BashComplete: func(c *cli.Context) {
+		commandoptions.CompleteFlags(commandoptions.CommandFlags(flagsAdopt, keysAdopt))
+	},
+}
+
+func flagsAdopt() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "[required] The stack name.",
+		},
+		cli.StringFlag{
+			Name:  "adopt-file",
+			Usage: "[required] Path to adopt stack data file",
+		},
+		cli.StringFlag{
+			Name:  "environment-file",
+			Usage: "[optional] File containing environment for the stack",
+		},
+		cli.StringFlag{
```

Comment:
@jrperritt since the api expects a string, I wonder if we need to change the `stack-adopt` (and other) request structure to use a string instead of an int?